File: Python/beginner/1864.py
# https://judge.beecrowd.com/pt/problems/view/1864
# Nossos Dias Nunca Voltarão

# Main Code
usrInput = int(input())

message = "LIFE IS NOT A PROBLEM TO BE SOLVED"

if usrInput > len(message):
    usrInput = len(message)

for i in range(usrInput):
    print(message[i], end="")
    
print()
    
# By: Fernando Serra
# https://github.com/fernandosserra

File: Haskell/beginner/1864.hs
-- https://judge.beecrowd.com/pt/problems/view/1864
-- Nossos Dias Nunca Voltarão

-- Main Code
main :: IO()
main = do

    usrInput <- readLn :: IO Int
    let citation = "LIFE IS NOT A PROBLEM TO BE SOLVED"

    putStrLn (take usrInput citation)
    
-- By Fernando Serra
-- https://github.com/fernandosserra    